source/stormwater.rst

```diff
@@ -0,0 +1,329 @@
+.. _stormwater:
+
+***********************************************
+Stormwater Retention Model
+***********************************************
+
+
+Overview
+========
+Planning agencies are increasingly considering urban water management in their strategies to address issues such as climate change, flood risk, or population growth, while protecting the environment. The InVEST stormwater runoff retention model can support this effort by providing information on two ecosystem services related to stormwater management: runoff retention, and groundwater recharge (Flood risk reduction is assessed in a separate InVEST model, :ref:`Urban Flood Risk Mitigation <ufrm>`). Runoff retention corresponds to the retention of stormwater by pervious land uses, which is beneficial given the detrimental effects of polluted stormwater discharge into rivers or the ocean. Groundwater recharge is a related service, corresponding to the percolation of stormwater past the root zone, potentially recharging groundwater for human and non-human purposes.
```

Comment:
Feedback from Milan:

> Considering the potential similarity between the stormwater model and the urban flood model, I feel like it would help to mention in the beginning of the model description maybe the main points in how they differ, to make the decision for which model to use easier for the potential end user.
This could for example include explaining why ‘retention costs’ can be calculated with the stormwater model but not with the flood risk model? Or why one model uses the Curve Number method, but the other uses Runoff Coefficients (which I can imagine users are likely to mix up if they use both models). Both questions I also still ask myself.

Depending how much there is to say, this could also go in a "Comparison to Urban Flood Model" section.